libraries/ESP8266WebServer/examples/HelloServer/HelloServer.ino

```diff
@@ -86,6 +86,74 @@ void setup(void) {
 
   server.onNotFound(handleNotFound);
 
+  /////////////////////////////////////////////////////////
+  // Hook examples
+
+  server.addHook([](const String & method, const String & url, WiFiClient * client, ESP8266WebServer::ContentTypeFunction contentType) {
+    (void)method;
+    (void)url;
+    (void)client;
+    (void)contentType;
```

Comment:
Minor style nitpick - by omitting argument name in the signature, there would be no need for these 4 lines.
There is also [`[[maybe_unused]]`](https://en.cppreference.com/w/cpp/language/attributes/maybe_unused) / `[[gnu::unused]]` for local declaration, since now we use c++17. But that one is mostly for stack variables